spec/models/order_balance_spec.rb

```diff
@@ -358,4 +392,42 @@
       end
     end
   end
+
+  context "with comparison operators" do
```

Comment:
Here as well. I fear that being a bit vague as opposed to clearly stating the method under test may cause these tests not to be thorough enough in the future.